Wiki/starting/starting-leaving.md

```diff
@@ -0,0 +1,146 @@
+# Leaving CLEX guidelines
+
+When leaving the Centre it is important to document and organise the data and code you are leaving beyond. This is both to ensure that unecessary files are removed and storage freed up in shared projects and that anything of value to your future self and/or others is properly stored and documented. 
+
+These guidelines are provided to cover the most common workflow. 
+
+You should not own any data on disk when you leave or shortly after you leave
+Any data you own on tape should be accessible for read and write by other people
+
+## The Exit Email
+
+About 2 months before you leave, you should receive the following email:
+
```

Comment:
I am not suggesting any changes to the email template as I am not sure you want me to do that, but the email text can definitely be improved.

Reply:
Stephen sends that, so unless there are major changes, it's not worth doing anything, at least for the moment.